vocode/streaming/synthesizer/cartesia_synthesizer.py

```diff
@@ -0,0 +1,67 @@
+import io
+import wave
+
+from vocode import getenv
+from vocode.streaming.models.message import BaseMessage
+from vocode.streaming.models.synthesizer import CartesiaSynthesizerConfig
+from vocode.streaming.synthesizer.base_synthesizer import BaseSynthesizer, SynthesisResult
+
+
+class CartesiaSynthesizer(BaseSynthesizer[CartesiaSynthesizerConfig]):
+    cartesia_tts = None
+
+    def __init__(
+        self,
+        synthesizer_config: CartesiaSynthesizerConfig,
+    ):
+        super().__init__(synthesizer_config)
+
+        # Lazy import the cartesia module
+        if CartesiaSynthesizer.cartesia_tts is None:
+            from cartesia.tts import AsyncCartesiaTTS
+            CartesiaSynthesizer.cartesia_tts = AsyncCartesiaTTS
+
+        self.api_key = getenv("CARTESIA_API_KEY")
```

Comment:
once `api_key` is apart of the config, could you make this a conditional (psudeo-code) of 
```
self.api_key = if config.api_key or getenv
if not self.api_key, throw error "no api key!"
```